single.php

```diff
@@ -1,8 +1,9 @@
 <?php if ( have_posts() ) {
 	while ( have_posts() ) :
 		the_post();
-		get_header();
-		if ( \PressbooksBook\Helpers\is_book_public() ) :
+        get_header();
+        $content_only = pb_content_only();
```

Comment:
create the content_only variable so it can be passed to the blade template later